packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -100,6 +100,20 @@ void main() {
     expect(sliderBox, paints..rect(color: customTheme.disabledActiveTrackColor)..rect(color: customTheme.disabledInactiveTrackColor));
   });
 
+  testWidgets('SliderThemeData assigns the correct default shapes', (WidgetTester tester) async {
+    final SliderThemeData sliderTheme = _createDefaultSliderTheme();
```

Comment:
This value should be `ThemeData().sliderTheme`.

Reply:
Done